packages/core/src/lib/util.ts

```diff
@@ -224,3 +225,23 @@ export function validatePackageId(input: string): string | null{
   }
   return null;
 }
+
+/**
+ * Removes a file or directory. If the path is a directory, recursively deletes files and
+ * directories inside it.
+ */
+export async function rmdirs(path: string): Promise<void> {
+  const stat = await fs.promises.stat(path);
+
+  // This is a regular file. Just delete it.
+  if (stat.isFile()) {
+    await fs.promises.unlink(path);
+    return;
+  }
+
+  // This is a directory. We delete files and sub directories inside it, then delete the
+  // directory itself.
```

Comment:
Why? What breaks if you just delete the directory?

Reply:
Node 12 has a recursive option when deleting directories. Node 10 doesn't, and rmdir fails.

Reply:
:facepalm: